### PR TITLE
[Fix] Minor fix for AMLC Tutorial - QA

### PR DESCRIPTION
**Pull request by sxjscience**
## Description ##
Some documentation improvement + Use dataclass

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: scripts/question_answering/squad_utils.py [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-import itertools
+import dataclasses
+from dataclasses import dataclass
 import re
 import numpy as np
-import numpy.ma as ma
 import warnings
 import os
-from tqdm import tqdm
+from tqdm.auto import tqdm
```
Comment (szha) [inline scripts/question_answering/squad_utils.py:10]: what's the difference?

Comment (szha) [inline scripts/question_answering/squad_utils.py:10]: interesting. I didn't realize that tqdm has a notebook mode.

**Review by szha — APPROVED** [review]
